tests/test_message.py

```diff
@@ -174,6 +174,24 @@ def test_split_long_unicode(self):
         ]
         assert chunk_message(msg, max_length=20) == expected
 
+    @pytest.mark.parametrize(
+        'max_length,message',
+        product(
+            [4, 5, 6, 7],
+            (
+                'øøøøøøøøøø',  # 2-byte
+                '。。。。。。。。。。',  # 3-byte
+                '💩💩💩💩💩💩💩💩💩💩',  # 4-byte
+            ),
+        ),
+    )
```

Comment:
I might have gone a little crazy on indentation here. Perhaps this would be nicer as multiple statements? I'm not sure.